actors/datacap/src/lib.rs

```diff
@@ -3,3 +3,4 @@
 
 pub mod v10;
 pub mod v11;
+pub mod v9;
```

Comment:
`fil_actor_interface` needs to be updated as well right?

Reply:
added datacap v9 to fil_actor interface